lib/index.js

```diff
@@ -85,25 +95,96 @@ Vero.prototype.track = function(payload, fn) {
  * @api public
  */
 
-Vero.prototype.alias = request('/users/reidentify', 'put');
+Vero.prototype.alias = request('alias', '/users/reidentify', 'put');
+
+/**
+ * Add or Remove Tags.
+ *
+ * http://developers.getvero.com/?bash#tags
+ *
+ * @param {Facade} msg
+ * @param {Object} tags
+ * @param {Function} fn
+ * @api public
+ */
+
+Vero.prototype.addOrRemoveTags = function(msg, tags, fn) {
```

Comment:
the part where you transform the payload should ideally be a mapper method in mapper.js and you can leave all the logic regarding the sending of the http request/`fn` stuff inside this function.